docs/platform/tutorials/image-processing/other-services.md

```diff
@@ -0,0 +1,195 @@
+# Other services
+
+There are some additional services in the pipeline that provide useful functionality. These range from S3 storage of data to calculation of the maximum vehicles per day in a specific location.
+
+![Other services](../image-processing/images/other-services-pipeline-segment.png)
+
+Briefly, these services are:
+
+1. *Cam vehicles* - calculates the total vehicles, where vehicle is defined as one of: car, 'bus', 'truck', 'motorbike'. This number is fed into the *Max vehicle window* service.
+
+2. *Max vehicle window* - calculates the maximum vehicles over a time window of one day. This service sends messages to the Data API service.
+
+3. *Data API* - this REST API service provide two endpoints: one returns the *Max vehicle window* values for the specified camera, and the other endpoint returns camera data for the specified camera. This API is called by the UI to obtain useful data.
```

Comment:
Just a thought, but I found my self wondering when it gets called. i.e. When you load or refresh the page or is it a poll mechanism? Might be useful contextual info.

Background: I was wondering why the front end needs a REST API to get the data when it can just retrieve it directly from the topics using SignalR. The chat app doen't need a REST API so why does this one need it?

Reply:
I thought it was for performance reasons to prevent the UI from being overloaded. I will do some more research and add some more detail - also related to Data Buffer service. TODO.